### ISSUE TITLE
[Question] will there be an SRT support ? 

### ISSUE THREAD

**Issue by SNRSE** [bbc/brave#67]
Hey i was wondering if there will be support for SRT as Input and Output ? 

I think Gstreamer already supports it

**Comment by matthew1000** [comment]
Yeah this would be great. Wouldn't be too hard. PRs most welcome. Unfortunately there's no active development on this project at the moment... something I'd love to change, but I just can't find time away from other projects at the moment.

**Comment by stemcc** [comment]
> Wouldn't be too hard. PRs most welcome.

Do you have any suggestions for where to start on this feature, or perhaps current code in this repository you can point to that could be similar in design to how the SRT feature might be implemented?

**Comment by moschopsuk** [comment]
https://github.com/bbc/brave/blob/master/brave/inputs/tcp_client.py

Would be a good place to start as an example.

The gStreamer docs are a good place to cover what properties can be set and the values needed.

https://gstreamer.freedesktop.org/documentation/srt/srtsrc.html
